tests/README.md

```diff
@@ -22,6 +22,41 @@ Each test vector is organized as follows:
 | `resolved-cri` | hex-encoded CBOR representation of the resolved CRI reference relative to the [base CRI](#bases) |
 | `resolved-uri` | resolved URI relative to the [base URI](#bases) |
 
+### Test Vector CSV Layout
+
+The test vectors are assembled in a CSV (RFC 4180)
+comma-separated-value file.
+
+Since `"` and `,` are very frequent, we instead use these settings:
+
+```ruby
+CSV.generate(col_sep: ";", quote_char: "|", quote_empty: false)
+```
+
+(Note that, with these settings, no quote character is used at all for
+the extant examples.)
+
+The columns are:
+
+| Key Name           | Description                                                                             |
+| ---                | ---                                                                                     |
+| `type`             | kind of test: `ok`, `rt` (round-trip), `red` (normalized on roundtrip); special value `base` |
+| `uri`              | a URI reference                                                                         |
```

Comment:
AIU this is optional; in cases when a CRI reference can not be expressed as a URI reference (or the test author didn't care about URIs in this scenario), it is often empty. Is this distinguished from the empty URI reference?